src/app/inbox/inbox-compose/inbox-compose.component.ts

```diff
@@ -7,6 +7,8 @@ import { PublicKeyService } from 'src/app/services/mailchain/public-key/public-k
 import { AddressesService } from 'src/app/services/mailchain/addresses/addresses.service';
 import { EnvelopeService } from 'src/app/services/mailchain/envelope/envelope.service';
 import { NameserviceService } from 'src/app/services/mailchain/nameservice/nameservice.service';
+import { BalanceService } from 'src/app/services/mailchain/addresses/balance.service';
```

Comment:
This also needs to be changed to a stub in `src/app/inbox/inbox-compose/inbox-compose.component.spec.ts` to prevent tests trying to call the real endpoints.
e.g.:
```
       { provide: BalanceService, useClass: BalanceServiceStub },
```

See other stubs for examples, to return expected values from the API without actually making the request.

Reply:
Currently I have a test failing:
`InboxComposeComponent ngOnInit initMail when composing a new message should initialize the model "from" field with currentAccount FAILED`